src/BannerAdSize.ts

```diff
@@ -43,10 +43,21 @@ export enum BannerAdSize {
   MEDIUM_RECTANGLE = 'MEDIUM_RECTANGLE',
 
   /**
-   * A (next generation) dynamically sized banner that is full-width and auto-height.
+   * @deprecated Use `ANCHORED_ADAPTIVE_BANNER` instead.
    */
   ADAPTIVE_BANNER = 'ADAPTIVE_BANNER',
 
+  /**
+   * A (next generation) dynamically sized banner that is full-width and auto-height.
+   */
+  ANCHORED_ADAPTIVE_BANNER = 'ANCHORED_ADAPTIVE_BANNER',
```

Comment:
Something occurred to me when I saw all these sizes, in our example app we are only every using ADAPTIVE_BANNER - https://github.com/invertase/react-native-google-mobile-ads/blob/eb149a62f26968c81646edd865567a9ee42936d6/RNGoogleMobileAdsExample/App.tsx#L162 - but we've code that branches on ad size, so it seems clear we should iterate on banner sizes as well in the App.tsx for example

Do you think it would be reasonable to have an array of all  banner sizes to show, then foreach banner size create + register one test component so each are visible? 

I wonder if that would have caught the fluid banner size issue on android actually! May be useful to actually draw a box with borderColor !== default and borderWidth >= 1 so it's clear whether it is showing or not, and my hypothesis is that an example iteration over all types would show the fluid issue, while demonstrating the three adaptive banner types work

What do you think?

Reply:
I think that's a good idea and added a test for every banner.